File: lib/src/main/java/com/sha/formvalidator/validator/DummyValidator.java
package com.sha.formvalidator.validator;

import android.widget.EditText;

/**
 * This is a dummy validator. It just returns true on each input.
 *
 */
public class DummyValidator extends Validator {

    public DummyValidator() {
        super(null);
    }

    public boolean isValid(EditText et) {
        return true;
    }
}

File: lib/src/main/java/com/sha/formvalidator/validator/DummyValidator.kt
package com.sha.formvalidator.validator

import android.widget.EditText

/**
 * This is a dummy validator. It just returns true on each input.
 *
 */
class DummyValidator : Validator("") {

    override fun isValid(et: EditText): Boolean {
        return true
    }
}
